src/collection-browser.ts

```diff
@@ -878,7 +913,7 @@ export class CollectionBrowser
       sortParam: this.sortParam ?? undefined,
       selectedSort: this.selectedSort,
       sortDirection: this.sortDirection ?? undefined,
-      selectedFacets: this.selectedFacets ?? defaultSelectedFacets,
+      selectedFacets: this.selectedFacets ?? getDefaultSelectedFacets(),
```

Comment:
One of the persistence issues was related to several different components blindly grabbing the _single_ `defaultSelectedFacets` object and mutating it. I've replaced it with a getter that returns a new object on every call.